docs/source/reference/builtin-devices.rst

```diff
@@ -40,32 +40,71 @@ This must be sub-classed (like :class:`~ophyd.device.Device`) to be useful.
    ophyd.epics_motor.EpicsMotor
    ophyd.epics_motor.MotorBundle
 
+.. _built_in.Epics.Scaler.record:
+
+EPICS Scaler
+------------
+
+The EPICS ``scaler`` record ([#scaler]_) is supported by two alternative
+``ophyd`` Devices.  (You need only choose one of these two.)  An important
+difference between the :ref:`built_in.EpicsScaler` and the :ref:`built_in.ScalerCH`
+Devices is in how each channel's name is represented, as summarized 
+in the next table:
+
+===========================  =======================================  ====================================================
+class                        channel naming                           examples
+===========================  =======================================  ====================================================
+:ref:`built_in.EpicsScaler`  numbered                                 ``scaler_channels_chan2``, ``scaler_channels_chan3``
+:ref:`built_in.ScalerCH`     EPICS scaler record channel name fields  ``I0``, ``diode``
```

Comment:
Should not those two refs point to the classes in https://github.com/bluesky/ophyd/blob/master/ophyd/scaler.py like in the example of EpicsMotor above?

Reply:
That's a good addition.

Reply:
The table will refer to the subsections.  Each subsection will point to its class API documentation which, in turn, provides a `[source]` link to a view of the Python code.

Reply:
@mrakitin Please check again.  Merge if OK.  Thanks!